modules/oadp-s3-compatible-backup-storage-providers.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
+
+:_content-type: CONCEPT
+[id="oadp-s3-compatible-backup-storage-providers_{context}"]
+= AWS S3 compatible backup storage providers
```

Comment:
The headings are a bit confusing to me. For example, this heading suggests that I'm going to get a list of compatible backup storage providers. However, the introductory sentence is a bit vague "variety of..." and "different backup" aren't specific at all. 

I also don't actually get any specifically supported backup storage providers in this section under this heading.

Reply:
Hi Joe, Thank you for your helpful feedback. I implemented many of your suggestions.

Reply:
@CarmiWisemon I agree with @JoeAldinger on the heading (even after some apparent edits). The heading introduces sections on supported and unsupported providers. Maybe:  _AWS S3 and backup storage providers_ or  _Understanding AWS S3 and backup storage providers_

Reply:
@mburke5678 
1. Thank you for pointing out the mention of s3 which needs to be corrected. I will open up a ticket requesting that it be corrected.
2.  The purpose of this topic based on the Jira ticket, is to explain which providers are compatible with AWS s3, which is why the topic is called "AWS s3 compatible backup storage providers".
3. Thank you for your feedback.
4. Please merge the PR as it is.
Thank you for your help!
Carmi